### DISCUSSION TITLE
TypeScript?

### DISCUSSION THREAD

**Discussion by iancanderson** [octokatherine/word-master#97] [Ideas]
👋 Thanks for this awesome repo!

I have a TypeScript branch that seems to be working well. I could make a PR if you'd like to convert word-master to TypeScript.

If you prefer to keep to as JS, that's cool too!

Here's the branch if you'd like to 👀 : https://github.com/octokatherine/word-master/compare/main...iancanderson%3Atypescript?body=&expand=1

**Comment by octokatherine** [comment]
yeesss @iancanderson 🙌 

update: merged!
